bokeh/tests/test_objects.py

```diff
@@ -181,6 +181,16 @@ def test_to_json(self):
                          (child_obj._id, obj._id),
                          json_string)
 
+    def test_no_units_in_json(self):
+        from bokeh.models import AnnularWedge
+        self.maxDiff = None
+        obj = AnnularWedge()
+        json = obj.to_json()
+        self.assertTrue('start_angle' in json)
+        self.assertTrue('start_angle_units' not in json)
+        self.assertTrue('outer_radius' in json)
+        self.assertTrue('outer_radius_units' not in json)
```

Comment:
Overall I see the point of this PR. Although in the case of `*_units` properties, I think this is only a temporary measure and there should be a special class of property to encode them.